crdt_commands_i_test.go

```diff
@@ -272,3 +272,77 @@ func TestUpdateAndFetchMap(t *testing.T) {
 		t.FailNow()
 	}
 }
+
+// UpdateHll
```

Comment:
This should be in a separate file with its own build constraint, something like this on the first line:

```golang
// +build integration_hll
```

That way it won't be run unless we mean it. An `integration-test-hll` target could then be added to `build/common.mk`